plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -144,7 +144,20 @@ export const OwnershipCard = ({
     error,
     value: componentsWithCounters,
   } = useAsync(async () => {
-    const entitiesList = await catalogApi.getEntities();
+    const kinds = ['Component', 'API'];
```

Comment:
Waiting for https://github.com/backstage/backstage/issues/4499 ...

We actually have an owner field on a lot more than these. Template, Resource, Domain and System right?

Reply:
Yeah, but the card only works for components and APIs right now. Would be cool if one could configure the card to include the things used in the specific org.